CHANGES_UNRELEASED.md

```diff
@@ -6,6 +6,13 @@
 
 <!-- WARNING: New entries should be added below this comment to ensure the `./automation/prepare-release.py` script works as expected.
 
+## Viaduct
+
+### What's New
+  - Allow viaduct to make requests to the android emulator loopback address via
+    a new viaduct_allow_android_emulator_loopback() (in Rust)/allowAndroidEmulatorLoopback() (in Kotlin)
+    ([#5270](https://github.com/mozilla/application-services/pull/5270))
+
```

Comment:
This should be moved below line 20 (outside of the `<!-- ... -->` comment block)